### ISSUE TITLE
util.js: Edge doesn't support property "rest" for objects.

### ISSUE THREAD

**Issue by jlith** [bestguy/sveltestrap#63]
While testing out the library I noticed that on Edge it didn't work, and traced the issue to the function "clean" and in it to "...rest", inside util.js:

```
  const { children, $$scope, $$slots, ...rest } = $$props; // TODO support keys
```

`So I changed it to this to get it working:

```
 const { children, $$scope, $$slots } = $$props;
 const rest = {};
 for (const key of Object.keys($$props)) {
   if (key !== "children" && key !== "$$scope" && key !== "$$slots") {
     rest[key] = $$props[key];
   }
 }
 return rest;
```

**Comment by bestguy** [comment]
Oh thanks, this should be transpiled in or your solution, will take a look thanks!

**Comment by bestguy** [comment]
Hi @jlith , thanks for the PR for this.  Would you mind trying installing and trying `sveltestrap@3.2.5-0`?   This version enables object spread in the transpilation.  I'll try later this weekend as well.

**Comment by jlith** [comment]
Hi! Installed that with "npm i sveltestrap@3.2.5-0". The util.js seems to be the same as previous? So it doesn't work in Edge.

**Comment by bestguy** [comment]
Thanks, just to confirm are you importing from the lib like:

`import { Card } from 'sveltestrap';`

or:

`import Card from 'sveltestrap/src/Card.svelte';`

?

If the second transpiling won't help, and we'd need your PR solution.  Importing from src requires the user or browser to support object spread.

If the first, my PR is probably incorrect and not transpiling correctly.

**Comment by jlith** [comment]
Like the 1st option: 
![image](https://user-images.githubusercontent.com/38461562/66333113-e9b97380-e93e-11e9-8de4-fb13b9277d78.png)

**Comment by bestguy** [comment]
Hi @jlith, your PR has merged and released with sveltestrap@3.2.5 (https://github.com/bestguy/sveltestrap/releases/tag/v3.2.5)

I was unable to figure out Rollup+Babel issues so thanks for the fix for now.
